src/components/Form/index.tsx

```diff
@@ -199,9 +257,11 @@ export const Form = ({
               error={Boolean(errors.claim)}
               helperText={errors.claim?.message}
             >
-              {inputOptions.claim.map((i: string) => (
+              {inputOptions.claim.map((i: string, index: number) => (
                 <MenuItem value={i} key={i}>
-                  {i}
+                  <Tooltip key={i} title={tooltips.claim[index]} placement='right' arrow>
+                    <div> {i}</div>
```

Comment:
The problem with this is, the tooltip is only over the text and not the whole menu item. To fix this problem, you can use a `Box` component and give it a width and height of 100% of its parent.